lwc/src/main/scala/quasar/physical/s3/S3Config.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.physical.s3
+
+import slamdata.Predef._
+import argonaut.{DecodeJson, DecodeResult}
+import org.http4s.Uri
+import cats.syntax.apply._
+import cats.syntax.flatMap._
+import cats.instances.option._
+import slamdata.Predef._
+
+final case class S3Config(bucket: Uri, parsing: S3JsonParsing)
+
+object S3Config {
```

Comment:
Could you make a comment in this file that shows an example config for each of the two modes (array and linedelim)?

Reply:
Or an alternative would be to have a `s3.config.example` file.